tests/foreman/ui_airgun/test_repository.py

```diff
@@ -131,6 +140,122 @@ def test_positive_create_as_non_admin_user(module_org, test_name):
             product.name, repo_name)[0]['Name'] == repo_name
 
 
+@tier2
+@upgrade
+def test_positive_create_puppet_repo_same_url_different_orgs(
+        session, module_prod):
+    """Create two repos with the same URL in two different organizations.
+
+    :id: f4cb00ed-6faf-4c79-9f66-76cd333299cb
+
+    :expectedresults: Repositories are created and puppet modules are visible
```

Comment:
`Repositories are created and have correct number of puppet modules synced` or any other proper description you like